legion/engine/physics/physics_contact.hpp

```diff
@@ -356,7 +379,7 @@ namespace legion::physics
 
 		void logRigidbodyState()
 		{
-			//log::debug("//--------logRigidbodyState----------//");
+			log::debug("//--------logRigidbodyState----------//");
```

Comment:
debug log

Reply:
the function is called logRigidbodyState() it does need the logging